src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,643 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:Avalonia.Controls"
+        xmlns:converters="using:Avalonia.Controls.Converters"
+        xmlns:primitives="using:Avalonia.Controls.Primitives"
+        xmlns:pc="clr-namespace:Avalonia.Controls.Primitives.Converters;assembly=Avalonia.Controls.ColorPicker"
+        xmlns:globalization="clr-namespace:System.Globalization;assembly=mscorlib"
+        x:CompileBindings="False">
```

Comment:
Note that I cannot use compiled bindings here as it throws errors. I didn't investigate this much.

Reply:
This mostly worked after #8466 I only had to add the x:DataType to the last nested style that uses the binding (the same one discussed above with the converter parameter). I didn't know that x:DataType could be added to Style itself actually.